newport/mockXPS.go

```diff
@@ -189,6 +200,15 @@ func (c *MockController) moveTo(axis string, pos float64) {
 			c.Unlock()
 			return
 		}
+
+		// Abort is Stop is called
```

Comment:
need to do all three of the map accesses inside the mutex-guarded period (lock, if, ...body..., unlock)